### ISSUE TITLE
Feat?Is it possible to add menu(TOC) support?

### ISSUE THREAD

**Issue by YuYoung32** [mivinci/hugo-theme-minima#56]
In markdown, it's called [TOC]. I tried {{TOC}} and [TOC], they both didn't work.

**Comment by mivinci** [comment]
Yeah actually I deleted TOC several commits back for some reasons and am considering adding it back currently.

**Comment by lin-snow** [comment]
同求Toc功能,这个在很长的文章里比较有用
